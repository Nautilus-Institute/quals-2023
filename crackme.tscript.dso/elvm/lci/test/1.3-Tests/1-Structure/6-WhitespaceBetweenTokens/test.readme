This test checks that whitespace in between tokens is handled properly.  It
tests whitespace only, tabs only, alternating whitespace and tabs, and
alternating tabs and whitespace.
