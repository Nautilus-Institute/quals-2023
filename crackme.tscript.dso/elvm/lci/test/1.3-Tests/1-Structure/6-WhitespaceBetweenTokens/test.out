Lorem ipsum dolor sit
Lorem ipsum dolor sit
Lorem ipsum dolor sit
Lorem ipsum dolor sit
